system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiClientTest.java

```diff
@@ -132,4 +157,20 @@ void getParticipant_notFound() {
                 .isEqualTo(404);
     }
 
+    private void assertIssuedVerifiableCredential(SignedJWT jwt, String clientDid, Instant startTime) throws ParseException {
+        assertThat(jwt.getJWTClaimsSet().getSubject()).isEqualTo(clientDid);
+        assertThat(jwt.getJWTClaimsSet().getIssueTime().toInstant()).isAfter(startTime);
+    }
+
+    @NotNull
+    private RegistryApi api(String did) {
```

Comment:
nit: should be called `createApi` or similar, to indicate what's being done. Same for `did()`.  
Also, those two could be moved to the `RegistrationServiceTestUtils` class as they are duplicated in the command line client test.

Reply:
renamed and moved to RegistrationServiceTestUtils